backend/msgpack/conn.go

```diff
@@ -60,7 +64,7 @@ type connection struct {
 	numFailures int
 	threshold   int
 
```

Comment:
Super nit: remove the blank line here. My personal preference is to arrange the fields in the following order:
```
type foo struct {
  embedded type

  consts / immutable input params

  internal fields / mutable input params

  testing vars / functions
}
```
Obviously you don't need to follow the same convention, but some kind of consistency would be good.